src/isMonday/index.ts

```diff
@@ -13,7 +13,7 @@ import requiredArgs from '../_lib/requiredArgs/index'
  *
  * - [Changes that are common for the whole library](https://github.com/date-fns/date-fns/blob/master/docs/upgradeGuide.md#Common-Changes).
  *
- * @param {Date|Number} date - the date to check
+ * @param {Date|Number} dirtyDate - the date to check
```

Comment:
Please leave it as is. In the code, the name makes sense because it highlights that the date is not converted yet (and not everywhere, here for example it could be simplidied), but in the docs, is just noise.

```suggestion
 * @param {Date|Number} date - the date to check
```